css/style.css

```diff
@@ -226,4 +226,71 @@ html {
 .btn--outline:active {
   background-color: rgba(0, 0, 0, 0.6);
   color: #f5f5f5;
+}
+
+/******************************/
+/* ABOUT SECTION */
+/******************************/
+
+.about-section {
+  background: url('../gallery/banners/about-banner.png');
+  background-size: cover;
+  padding-top: 7.6rem;
+  padding-left: 29rem;
+  width: 160rem;
+  height: 71rem;
+  display: inline-flex;
+  justify-content: flex-start;
+  box-shadow: inset 0 0 6rem rgb(0, 0, 0, 0.1);
+  position: relative;
+}
+
+.about-section:before {
+  content: '';
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to top left, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+}
+
+.our-mission {
+  position: relative;
+  width: 42rem;
+  padding-top: 2.2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
```

Comment:
If you're using custom margins on container items, `gap` here is not expected to be used